_includes/scenarios/integrate/requests-over-templates.md

```diff
@@ -102,13 +107,19 @@ If you want to reject the Request you can do so by calling the `PUT /api/v2/Requ
 }
 ```
 
-In the response you can see the Request has moved to status `Decided`. This is where the enmeshed Runtime steps in and handles the Request based on you decision. Because you rejected the Request, the enmeshed Runtime will only move the Request to status completed. This behavior can be observed by querying the Request again after a few seconds (`GET /api/v2/Requests/Incoming/{id}`).
+In the response you can see that the Request has moved to `status` `"Decided"`.
+This is where the enmeshed Runtime steps in and handles the Request based on you decision.
+It creates a [Response]({% link _docs_integrate/data-model-overview.md %}#response) with the appropriate [RejectResponseItems]({% link _docs_integrate/data-model-overview.md %}#rejectresponseitem) and returns it to the Sender of the Request.
+Then, it will move the Request to `status` `"Completed"`.
+This can be observed by [querying the Request]({% link _docs_use-cases/use-case-consumption-get-incoming-request.md %}) again after a few seconds.
 
 ### Accept
 
-If you tried out the Rejection before this step make sure to create a Request by [loading the Template again](#load-the-template-and-get-the-request) with the same truncated reference.
+If you tried out the rejection before this step, make sure to create a new Request by [loading the Template](#load-the-relationshiptemplate-and-get-the-request) again with the same `truncatedReference`.
```

Comment:
We should write "RelationshipTemplate" here instead of "Template".

The same applies to another location and to the title of the scenario. I'd rename the scenarios to "Requests via RelationshipTemplates" and "Requests via Messages".

Reply:
As discussed, the renaming will be conducted in a separate PR.